vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -60,13 +60,17 @@ A cache issue impacting the first deployment of the `entando-hub-content-bundle`
 
 To clear the cache, select `Administration` from the bottom of the left menu, then `Reload configuration`.
 
-Alternatively, restarting the quickstart-server pod (which contains the Entando App Engine) will also clear the cache, and can be achieved with `ent k delete pod/<YOUR QUICKSTART-SERVER POD>`, e.g. `ent k delete pod/quickstart-server-deployment-5d785b997c-r4sc8`. It will take several minutes for the pod to redeploy after deletion. 
+Alternatively, restarting the quickstart-server pod (which contains the Entando App Engine) will also clear the cache, and can be achieved with `ent k delete pod/YOUR-QUICKSTART-POD-NAME`, e.g. `ent k delete pod/quickstart-deployment-5d785b997c-r4sc8`. It will take several minutes for the pod to redeploy after deletion. 
```

Comment:
No need to change this now but I think we can remove all of the "clear the cache" language. I added a note to ENDOC-361. We just need to test a little first.